projects/storefrontapp-e2e-cypress/cypress/helpers/coupons/my-coupons.ts

```diff
@@ -100,6 +116,7 @@ export function claimCoupon(couponCode: string) {
     'getClaimedCouponPage'
   );
 
+  //TODO when 'enableClaimCustomerCouponWithCodeInRequestBody' is true, call the mothod of 'waitForClaimCouponWithCodeInBody' instead once ClaimCustomerCouponWithCodeInBody works in the backend(available since Commerce 2211.28)
```

Comment:
Hi @rmch91 , same above with how to update old case properly after enable toggle.
Thanks.